components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -161,4 +161,16 @@
     <string name="mozac_feature_addons_unsupported_caption_plural">%1$s add-ons</string>
     <!-- Text link to a sumo page for learning more about unsupported add-ons. -->
     <string name="mozac_feature_addons_unsupported_learn_more">Learn more</string>
+    <!-- Text that describes the last known status of the add-on updater - this status indicates when an add-on was successfully updated  -->
+    <string name="mozac_feature_addons_updater_status_successfully_updated">Successfully updated</string>
+    <!-- Text that describes the last known status of the add-on updater - this status indicates when the updater didn't find a new update available -->
+    <string name="mozac_feature_addons_updater_status_no_update_available">No update available</string>
+    <!-- Text that describes the last known status of the add-on updater - this status indicates when an error happened while updating. -->
+    <string name="mozac_feature_addons_updater_status_error">Error</string>
+    <!-- Text shown in a dialog that displays information about the last intent to update an add-on. This is the title of the dialog. -->
```

Comment:
Pretty sure you mean "attempt" in these 3 comments?

> Title of the dialog displaying information for the last attempt to update add-ons.

Reply:
Thanks 👍